### ISSUE TITLE
Allow faster updated from ORCID

### ISSUE THREAD

**Issue by nemobis** [dissemin/dissemin#448]
ORCID titles are cached, I don't know for how long; it would be nice to let the authenticated user refresh them. For instance here we now have 64 known works (thanks "claim" button in BASE!) but Dissemin still displays just 6.
https://dissem.in/r/71492/mark-rubin
https://orcid.org/0000-0002-6483-8561

**Comment by RaitoBezarius** [comment]
AFAIK, there are no "automatic ORCiD" updates, we cache ORCiD papers on demand (you have a "Refresh my publications" button on your profile which will trigger a refresh from ORCiD).
Though, we could be able to add some kind of cron task to refresh publications periodically or something like that.

**Comment by nemobis** [comment]
Ah! I had not noticed. For myself I see "Refresh publications" from https://dissem.in/r/5703/federico-leva and https://dissem.in/my-profile , this might be enough for the use case "I've just updated ORCID and want to see the data in Dissemin".

A regular refresh makes sense, how long does it take to updated all profiles?

**Comment by wetneb** [comment]
yeah it totally makes sense. Actually I want to prefetch all orcid profiles with the public dump. But it does take some time to fetch profiles. I suppose we could at least refresh once a month.

I think there should also be possible for normal users to refresh the publications of any profile.

**Comment by nemobis** [comment]
Antonin Delpeuch, 03/09/2017 17:34:
> yeah it totally makes sense. Actually I want to prefetch all orcid 
> profiles with the public dump. But it does take some time to fetch 
> profiles. I suppose we could at least refresh once a month.

Once a month would be good!
